### DISCUSSION TITLE
[New Device Support]: Nokia Streaming Box

### DISCUSSION THREAD

**Discussion by ChristophCaina** [PRProd/HA-Firemote#76] [New Device Family Support Request]
### Device Name

Nokia Streaming Box 8000

### Which compatability mode event number works BEST with your device?

None

### Additional Comments (optional)

should be similar to SHIELD - it is a basic Android TV ...
Currently, Shield seems to work pretty fine - it would only be nice to have a matching image of the remote controller :)
[I don't know how / where I can create the image resources]

![image](https://user-images.githubusercontent.com/26391061/227999944-bb76e130-9844-428b-8614-bae05f88ccb4.png)

**Comment by PRProd** [comment]
I will add this to my list!

Please note: It might take a while for me to get to this one because the device is not easily available in my country, (US)  As soon as I get one, I will begin the process.

**Comment by ChristophCaina** [comment]
Hi, Thank you :)

I might be able to support on this, once I figured out how this all here is working ;)
From what I noticed so far, the Remote from the Nvidia Shield is pretty much working in all cases I've tested so far (except the emby app ^^)
So I guess, the main task would be to create the Layout for the Remote Control...

... and... when I think about it, having the "Google Voice button" linked to open either the Assist from HA or on the Companion App the Google Voice Command ;)

One Idea:
Maybe just add the layout for the remote and create a copy of the Shield Pro (2019)... then, we could start figuring out if some edge cases might not work?

**Comment by PRProd** [comment]
@ChristophCaina - I'm going to close this one temporarily.  I'll open it back up once I start working on official support.

Also, I re-built the poll over at #41 which deleted everyone's responses, so you might want to head over there and vote again.  :smile:

**Comment by PRProd** [comment]
The poll over at https://github.com/PRProd/HA-Firemote/discussions/41 has been reset once again after the latest release which added Chromecast support, which means that your previous vote has been removed.  If you're still interested in support for this one, I recommend going back and voting again.

**Comment by PRProd** [comment]
@ChristophCaina - Apple TV won the last round, so after that release the poll over at https://github.com/PRProd/HA-Firemote/discussions/41 has been reset once again. If you're still interested in Firemote support for the Nokia Streaming box, I recommend going back and voting again.

**Reply by ChristophCaina** [reply]
Thanks.
I voted again :)
But the streaming Box is already working pretty Well with the current Versions.

Don't know, If there are some Special Things that don't Work :)
